src/Microsoft.AspNet.Server.Kestrel/Infrastructure/MemoryPoolIterator2.cs

```diff
@@ -9,6 +9,8 @@ namespace Microsoft.AspNet.Server.Kestrel.Infrastructure
 {
     public struct MemoryPoolIterator2
     {
+        private readonly static int _vectorSpan = Vector<byte>.Count;
```

Comment:
`Vector<byte>.Count` should Jit to const when intrinsic; not sure when not `Vector.IsHardwareAccelerated == false` ; however we know `readonly static int` does Jit to const.